xml/System/FlagsAttribute.xml

```diff
@@ -52,7 +52,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Indicates that an enumeration can be treated as a bit field; that is, a set of flags.</summary>
+    <summary>Ensures that ToString() will return well formatted combinations of enum values where an enumeration is treated as a bit field.</summary>
```

Comment:
I don't think this is a good change. While it's (as far as I remember) true that `System.Enum` is largely only impacted on `ToString`, this flag is intended to be general metadata that other things can and may depend on to understand that `Flag1 | Flag2` is "valid", and not just unknown or undefined behavior.

Reply:
For example, `System.ComponentModel.Annotations` checks for this flag to customize its own behavior on repporting whether a value is valid.